src/sagemaker/fw_utils.py

```diff
@@ -638,6 +639,36 @@ def warn_if_parameter_server_with_multi_gpu(training_instance_type, distribution
         logger.warning(PARAMETER_SERVER_MULTI_GPU_WARNING)
 
 
+def profiler_config_deprecation_warning(
```

Comment:
nit: if we can utilise any of the useful offerings in https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/deprecations.py

Reply:
Absolutely, will do, will have to add some utils that allow me to "freeform" the message, as opposed to existing format. But yes.